docs/source/index.rst

```diff
@@ -135,6 +135,7 @@ To install all extra modules, use the ``all`` extra.
    integrations/integration-sklearn
    logging-examples/logging-feature-plots
    logging-examples/visualizing-logged-dataframes
+   logging-examples/multiple-backend
```

Comment:
last thing - I'm just being nitpicky - can you flip these so the titles on the sidebar are alphabetical?

Reply:
Hahah no that's all good! On it